google-cloud-spanner/pom.xml

```diff
@@ -191,6 +191,11 @@
       <groupId>io.opencensus</groupId>
       <artifactId>opencensus-contrib-grpc-util</artifactId>
     </dependency>
+    <dependency>
```

Comment:
I don't think we should add the implementation here. In fact this is something that the user application should do.

Reply:
Views are not being populated in the unit tests without this dependency. The scope is adjusted appropriately.

Reply:
Another option is to mock OpenCensus implementation.